docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -17,6 +17,19 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Distributed Tracing in Grafana Cloud k6
+
+<!-- Heitor Tashiro Sergent -->
+<!-- OSS, Enterprise -->
```

Comment:
Is this just something left over or some type of tag?  The feature is only available in Grafana cloud - specifically for customers visualizing results in GCk6 and using GCT

Reply:
Copy+paste that I didn't pay attention to 😓 , thanks for catching that. 🙏